bitnami/elasticsearch/Chart.yaml

```
@@ -34,4 +34,4 @@ maintainers:
 name: elasticsearch
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/elasticsearch
-version: 21.2.8
+version: 21.3.8
```

Patch number should be reset to 0 on minor versions:

```suggestion
version: 21.3.0
```